pallets/gear/src/mock.rs

```diff
@@ -29,7 +29,11 @@ use sp_runtime::{
 type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
 type Block = frame_system::mocking::MockBlock<Test>;
 
-pub const BLOCK_AUTHOR: u64 = 255;
+pub(crate) const BLOCK_AUTHOR: u64 = 255;
+pub(crate) const USER_1: u64 = 1;
+// todo [sab] change accounts order + change names to MSG_SENDER and etc.
+pub(crate) const LOW_BALANCE_USER: u64 = 2;
+pub(crate) const USER_2: u64 = 3;
```

Comment:
I would suggest more conventional names for cryptography like Alice, Bob & their friends

Reply:
The fact is that we need to state in user name, that he has low balance, that's why `LOW_BALANCE_USER` was introduced. Some tests check balance stuff, so such user is needed. IMHO, `Alice`, `Bob`, `BLOCK_AUTHOR` and  `LOW_BALANCE_USER` is no difference with `USER_1`, `USER_2` and etc.

What do you think?